source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+.. include:: ../references.txt
+
+.. iact_events:
```

Comment:
You changed the label from `iact-event` to `iact_events` and now there's three broken links to this section:
```
/Users/deil/code/gamma-astro-data-formats/source/data_storage/hdu_index/index.rst:67: WARNING: undefined label: iact-events (if the link has no caption the label must precede a section header)
/Users/deil/code/gamma-astro-data-formats/source/info/time.rst:143: WARNING: undefined label: iact-events (if the link has no caption the label must precede a section header)
/Users/deil/code/gamma-astro-data-formats/source/ogip/index.rst:8: WARNING: undefined label: iact-events (if the link has no caption the label must precede a section header)
```
Please fix.
(I don't care what the label is, either version is fine)